Website/src/rendering/src/components/Speakers/FeaturedSpeakers.tsx

```diff
@@ -24,26 +35,31 @@ const FeaturedSpeakers = (props: FeaturedSpeakersProps): JSX.Element => {
     props.fields.data.item.children.results
       .filter((item) => item.featured.value)
       .sort()
-      .slice(0, parseInt(props.params.NumberOfSpeakers))
+      .slice(0, getSpeakerNumberToShow(props))
       .map((speaker, index) => (
         <Link key={index} href={`/speakers/${speaker.name.value}`} passHref>
-          <a className="grid-item">
-            <Image
-              field={speaker.picture.jsonValue}
-              alt={speaker.name}
-              width={265}
-              height={265}
-              loading="lazy"
-            />
-            <div className="item-details">
-              <Text tag="p" field={speaker.name} />
+          <a>
+            <div className="grid-item">
+              <div className="item-image">
+                <Image
+                  field={speaker.picture.jsonValue}
+                  alt={speaker.name.value}
+                  width={265}
```

Comment:
It feels weird on the speakers listing page on mobile that the featured speakers images are smaller than the regular speakers images.

![image](https://user-images.githubusercontent.com/2000200/151561442-a93ba767-52c4-4d0e-89a9-b285bb460669.png)